index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="ja">
+<head>
+    <meta charset="UTF-8">
+    <title>JavaScript Lesson1-2</title>
+</head>
+<body>
+    <script>
+      while(true){
+        let capital = window.prompt('日本の首都は？');
```

Comment:
`capital` に対して再代入を行わないので `const` でよいかと思いました。

```suggestion
        const capital = window.prompt('日本の首都は？');
```